src/ogma/index.ts

```diff
@@ -305,10 +311,21 @@ export class LKOgma extends Ogma<LkNodeData, LkEdgeData> {
   /**
    * Do a full reset on ogma and streams of ogma
    */
-  public shutDown() {
+  public shutDown(): void {
     this.destroy();
     if (this.store) {
       this.store.clear();
     }
   }
+
+  /**
+   * Reset the Ogma instance so that it can be used fresh in the next visulization
+   */
+  public resetOgmaLKE(): void {
```

Comment:
why not calling this method: `clearOgmaState`? That would be more representative of what the method is actually doing.